### ISSUE TITLE
IllegalArgumentException when trying to ListMergeRequests

### ISSUE THREAD

**Issue by yoelgluschnaider** [PavlikPolivka/GitLabProjects#49]
Running "List Merge Requests" in Android Studio and get the following exception:

```
Argument for @NotNull parameter 'mergeRequestListWorker' of com/ppolivka/gitlabprojects/merge/list/GitLabMergeRequestListDialog.<init> must not be null
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'mergeRequestListWorker' of com/ppolivka/gitlabprojects/merge/list/GitLabMergeRequestListDialog.<init> must not be null
	at com.ppolivka.gitlabprojects.merge.list.GitLabMergeRequestListDialog.$$$reportNull$$$0(GitLabMergeRequestListDialog.java)
	at com.ppolivka.gitlabprojects.merge.list.GitLabMergeRequestListDialog.<init>(GitLabMergeRequestListDialog.java)
	at com.ppolivka.gitlabprojects.merge.list.GitLabMergeRequestListAction.apiValidAction(GitLabMergeRequestListAction.java:23)
	at com.ppolivka.gitlabprojects.common.GitLabApiAction.actionPerformed(GitLabApiAction.java:58)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:191)
	at com.intellij.ide.actions.GotoActionAction$7.run(GotoActionAction.java:275)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:345)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:329)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:857)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:658)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:386)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

```
Android Studio 2.2.3

GitLab hosted internally in the company. Put the base server URL and the private token from the Profile Settings -> Account Tab.

**Comment by PavlikPolivka** [comment]
Strange I will look into this.

**Comment by yoelgluschnaider** [comment]
When this issue occurred I actually tried only with the host URL. With the full URL that I got from `git remote -v` It failed as well (no crash though) giving me a strange error: 

> Cannot Create Merge Request
Though I ran "List Merge Requests".

**Comment by PavlikPolivka** [comment]
Should be fixed by #69.
